cockroachcloud/create-your-cluster.md

```diff
@@ -27,11 +27,9 @@ To create and connect to a 30-day free {{ site.data.products.dedicated }} cluste
 
 In the **Cloud provider** section, select either **Google Cloud** or **AWS** as your preferred cloud provider.
 
-{{ site.data.products.db }} GCP clusters use [N2 standard](https://cloud.google.com/compute/docs/machine-types#n2_machine_types) machine types and [Persistent Disk storage](https://cloud.google.com/compute/docs/disks#pdspecs). AWS clusters use [M5 instance types](https://aws.amazon.com/ec2/instance-types/m5/#Product_Details) and [Elastic Block Store (EBS)](https://aws.amazon.com/ebs/features/). The IOPS associated with each node size in GCP is equal to 30 times the storage size, and the IOPS for AWS nodes is equal to 15 times the storage size.
+{{ site.data.products.db }} GCP clusters use [N2 standard](https://cloud.google.com/compute/docs/machine-types#n2_machine_types) machine types and [Persistent Disk storage](https://cloud.google.com/compute/docs/disks#pdspecs). AWS clusters use [M5 instance types](https://aws.amazon.com/ec2/instance-types/m5/#Product_Details) and [Elastic Block Store (EBS)](https://aws.amazon.com/ebs/features/). 
```

Comment:
For now, they will actually be using n1-standard machine types. We weren't able to get the n2 machine types in just yet. That will need to be an update later on.

Reply:
Is the pricing I have here still accurate?

Reply:
That's a good call, prices have actually changed. I'll slack you to get your thoughts on something, but here's the latest pricing for reference (with a calculator tab for combinations): https://docs.google.com/spreadsheets/d/1FOd6CC6ayFBec3KFwG2stgvYjQymkkqksx39XfuL2tk/edit?usp=sharing